services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -245,12 +245,21 @@ async def update_me(self, profile_update: ProfileUpdate) -> Profile:
 
     @_exception_mapper({})
     async def create_project(
-        self, project: ProjectCreateNew, *, is_hidden: bool
+        self,
+        project: ProjectCreateNew,
+        *,
+        is_hidden: bool,
+        parent_project_uuid: ProjectID | None,
+        parent_node_id: NodeID | None,
     ) -> ProjectGet:
         # POST /projects --> 202 Accepted
         response = await self.client.post(
             "/projects",
             params={"hidden": is_hidden},
+            headers={
+                "X-Simcore-Parent-Project-Uuid": f"{parent_project_uuid}",
```

Comment:
Create constants for these keys ... otherwise they are easy to flip a letter ...

Reply:
done